media-player.html

```diff
@@ -63,4 +66,6 @@
       });
     })();
   </script>
-</html>
+</body>
+
+</html>
```

Comment:
I'm not a professional web dev... is it normal to leave the ending newline off of files? That seems odd to me, especially since you're adding other newlines above. If you could, resave this file with the ending newline intact.

Reply:
Weird.. But yeah, when I try to save with a newline at the end, vs code strips it off..
Dunno why it does it, but seems intentional.
I resaved with another editor to keep the newline.